tests/test_jax_nonbonded.py

```diff
@@ -0,0 +1,151 @@
+import jax
+jax.config.update("jax_enable_x64", True)
+
+import numpy as onp
+from numpy.random import randn, rand, randint, seed
+seed(2021)
+
+from jax import numpy as np, value_and_grad, jit, vmap
+
+from jax.ops import index_update, index
+from timemachine.potentials.nonbonded import nonbonded_v3, nonbonded_v3_on_specific_pairs
+from timemachine.potentials.jax_utils import convert_to_4d, get_all_pairs_indices, get_group_group_indices
+
+from functools import partial
+from typing import Tuple, Callable
+
+Conf = Params = Box = ChargeMask = LJMask = LambdaPlaneIdxs = LambdaOffsetIdxs = np.array
+Lamb = Beta = Cutoff = Energy = float
+
+nonbonded_args = Conf, Params, Box, Lamb, ChargeMask, LJMask, Beta, Cutoff, LambdaPlaneIdxs, LambdaOffsetIdxs
+NonbondedArgs = Tuple[nonbonded_args]
+NonbondedFxn = Callable[[*nonbonded_args], Energy]
+
+
+def generate_random_inputs(n_atoms: int, dim: int = 3) -> NonbondedArgs:
+    # jittered
+    offsets = onp.array([onp.arange(n_atoms)] * dim).T
```

Comment:
Whats the need for these offsets in generating the conformation?

Reply:
Clarified in https://github.com/proteneer/timemachine/pull/453/commits/cb49add53a032f0671e82f98ce9de54ebf216217 -- goal is to avoid generating a random configuration where a pair of points is right on top of each other. There are more direct ways to do this

Reply:
Thanks for clarifying - I think this is fine then.